Design/DataProcessorAlgorithmUI/DataProcessingUserInterface.md

```diff
@@ -60,11 +97,69 @@ Many technique areas have now developed DataProcessorAlgorithms, as this is now
 The high-level solution involves refactoring and further generalizing the Reflectometry Reduction Interface into a **Data Processor User Interface**.
 
 **Key solution features**
-* The bulk of the solution will be about generating a `QWidget` subclass called `DataProcessorAlgorithmWidget`  
-* `DataProcessorAlgorithmWidget`  will take the name of the `DataProcessorAlgorithm` as one of its construction arguments
-* The `DataProcessorAlgorithmWidget` will provide virtual functions for overriding a `preProcess` and `postProcess` 
-* We also need a way to customise the output table in circumstances that direct mapping between `DataProcessorAlgorithm` properties and the viewable batch Table do not make sense. Currently `QReflTableModel` is specifically set up to do this for the `ReflectometryReductionOneAuto` DataProcessorAlgorithm.
-* 
+* The deliverable will be a `QWidget` subclass called `DataProcessorAlgorithmWidget`  
```

Comment:
When you say that the deliverable will be `DataProcessorAlgorithmWidget` I assume we mean that this will still be an assemblage of other widgets extracted from the reflectometry interface so that they can be reused elsewhere?